lib/rubocop/cop/rails/avoid_duplicate_ignored_columns.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # Checks for duplicate calls to ActiveRecord::Base.ignore_columns= as the
+      # subsequent calls will override the first call.
+      #
+      # @example
+      #   # bad
+      #   class User < ApplicationRecord
+      #     self.ignored_columns = %w(profile)
+      #     self.ignored_columns = %w(birthday)
+      #   end
+      #
+      #   # good
+      #   class User < ApplicationRecord
+      #     self.ignored_columns = %w(profile birthday)
+      #   end
+      #
+      class AvoidDuplicateIgnoredColumns < Base
```

Comment:
How about a name like this?

```suggestion
      class DuplicateIgnoredColumns < Base
```